docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md

```diff
@@ -17,7 +17,10 @@ options.addPreference("network.proxy.type", 0);
 driver = new RemoteWebDriver(options);
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR
+from selenium.webdriver.firefox.options import Options
+options = Options()
+options.add_argument("--headless")
```

Comment:
we should show use of the `headless` property instead... like this: 
`options.headless = True`

Reply:
Thanks for comment. I have updated the code with above suggestion